### ISSUE TITLE
errors on importing seed data

### ISSUE THREAD

**Issue by 12ian34** [commercelayer/commercelayer-seeder-cli#3]
Hello, 

On running your seed import into my brand new, empty test environment using `commercelayer-seeder -i $CL_CLIENT_ID -s $CL_CLIENT_SECRET -e https://<subdomain>.commercelayer.io`, and then viewing the import status at https://<subdomain>.commercelayer.io/admin/settings/imports , I see that there are many errors:
![image](https://user-images.githubusercontent.com/9979944/114593746-5922c880-9c84-11eb-8cd9-999df95bc9a0.png)

How can I go about checking these errors?

Many thanks,

**Comment by acasazza** [comment]
Hi @12ian34, 
What application are you using? Maybe it could not have the right permissions to create SKUs.

**Comment by 12ian34** [comment]
Hi @acasazza . I am using the seeder through an "integration" application with "Role": "Admin". I don't think the issue is with creating skus, since it managed to create 100 skus, which I notice is the Trial account type limit so I think that's the issue.

**Comment by acasazza** [comment]
It could be for that, btw, you can create a new organization and setting 50 or 90 items with `-m, --maxItems` when you start the seeder again. Or delete SKUs by API.

**Comment by 12ian34** [comment]
Indeed, thank you. I'd imagine that most of the users using your seed data are using it in a Trial license environment, so, I would recommend one of the following courses of action to improve usability of the seeder:
- provide a clearer notice in the seeder readme that the Trial license has a limit of 100 skus and the default command provided will result in errors
- halve the sku count of the seed data to fit nicely within the Trial.

**Comment by acasazza** [comment]
Sure! I will update the `README.md` with a note. Thanks so much for your advice. Much appreciate!

**Comment by mountmike** [comment]
I had a similar issue/error when following the onboarding docs. 

I fixed running `npm install -g @commercelayer/commercelayer-seeder-cli`
